composer.json

```diff
@@ -43,7 +38,12 @@
       "@lint-php"
     ],
     "test": [
-      "phpunit --coverage-text"
+      "phpunit --coverage-text",
+      "php local/scripts/make-clover-relative.php ./tests/reports/clover.xml"
```

Comment:
This converts all file paths inside the coverage reports to be relative to the project root instead of absolute paths inside the Docker container which was causing issues to Coveralls reporting.